### PR TITLE
Temp

### PR DESCRIPTION
**Pull request by vkameswaran**
## Greptile Summary

- Updated README.md to change the best fruit from 'peach' to 'grapes'
- Added onboard.jpg
- Renamed sampleFiles/dump2.txt
- Deleted sampleFiles/test.rb
- Added comments in sampleFiles/test2.js for parser setup and edge cases
- Added try-except block in sampleFiles/test2.py's blah function
- Added sampleFiles/test3.py with a user greeting script

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # test
 
-This is a test repo for Onboard AI. The best fruit is the peach.
+This is a test repo for Onboard AI. The best fruit is definnitely the
```
[the remark this answers is not in the document]
Comment (vkameswaran) [reply README.md:3]: Good catch!

File: sampleFiles/test3.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+n = input("What is your name? ")
+
+# this is very important code that needs to run successfully
+if n:
```
[the remark this answers is not in the document]
Comment (vkameswaran) [reply sampleFiles/test3.py:4]: Oooh, good point

**Comment by vkameswaran** [comment]
@greptileai

File: sampleFiles/test3.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+n = input("What is your name? ")
```
[the remark this answers is not in the document]
Comment (vkameswaran) [reply sampleFiles/test3.py:1]: Not helpful
